.github/workflows/wkorg-nightly.yaml

```
@@ -17,7 +17,7 @@ jobs:
 
       - uses: actions/setup-node@v4
         with:
-            node-version: 18
+            node-version: 22
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)